tox.ini

```diff
@@ -19,6 +19,7 @@ envlist =
     {pypy,py2.7}-django-1.6
 
     {pypy,py2.7,py3.5,py3.6,py3.7,py3.8}-flask-{1.0,0.11,0.12,dev}
+    {pypy,py2.7,py3.5,py3.6,py3.7,py3.8}-bottle-{0.12.13,0.12.14,0.12.15,0.12.16}
```

Comment:
I would prefer to only test `0.x` versions, not every bugfix release.

Reply:
No problem. Should be fixed by now.